src/main/kotlin/com/terraformation/backend/accelerator/variables/AcceleratorProjectVariableValuesService.kt

```
@@ -55,10 +55,7 @@ class AcceleratorProjectVariableValuesService(
 
   private val variablesById: Map<VariableId, Variable> by lazy {
     projectAcceleratorVariablesStableIds
-        .map {
-          variableStore.fetchByStableId(it.value)
-              ?: throw IllegalStateException("No variable with stable ID ${it.value}")
-        }
+        .mapNotNull { variableStore.fetchByStableId(it.value) }
```

Curious what motivated this change. The original version was designed to fail fast if one of the expected variables wasn't defined. With this change, you can no longer be certain that `variablesById` has an ID for every one of the hardwired stable IDs.

---

I am thinking whether we want our software to fail, when the variable list is not updated. 

I think returning `null` and logging the problem is likely a better solution for now.

---

But I am also happy to revert this if failing loudly is the preferred approach.